src/htrun/detect/__init__.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2018, Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""! @package mbed-ls
```

Comment:
Why are we bringing in the mbed-ls code? I thought we were leaving mbed-ls where it was and just specifying it as a package dependency?

Reply:
To make `htrun` complete standalone thought to copy `detect` locally in this repo from `mbed-os-tools/src/mbed-os-tools` as `detect` doesn't have any dependency with `mbed-os-tools` or `mbed-ls` (abstract layer which consumes the mbed-os-tools detect) but I have corrected with package dependency with `mbed-ls` instead of keeping a local copy of `detect`